### ISSUE TITLE
SubscriberClient authorisation

### ISSUE THREAD

**Issue by vie10** [crackthecodeabhi/kreds#8]
How to auth the SubscriberClient? It doesn't implement auth() method

**Comment by crackthecodeabhi** [comment]
@vie10 good catch, need to add auth mechanism for subscriber clients. Will be added in 0.8.
